### ISSUE TITLE
Extract agent/EV coordinates from the data for placement over SLD and map SVGs

### ISSUE THREAD

**Issue by tsmbland** [ImperialCollegeLondon/gridlington-vis#74]
We want to take agent/EV location data in the Opal and/or DSR dataframes and convert this to x/y coordinates for placement over the map and SLD [SVGs](https://github.com/ImperialCollegeLondon/gridlington-vis/issues/35#issuecomment-1799282195)

See https://github.com/ImperialCollegeLondon/gridlington-vis/issues/35#issuecomment-1768248996

There are four functions that need to be modified (they are just making up random data for now):

`get_agent_sld_coordinates`
`get_ev_sld_coordinates`
`get_agent_map_coordinates`
`get_ev_map_coordinates`

More details about what these functions should be doing can be found in their docstrings

**Comment by AdrianDAlessandro** [comment]
@tsmbland Can you just double check the description in this issue is up to date?

**Comment by pclemow** [comment]
No, I dont expect I will have time to get that working before Christmas.
